modules/standard/Types.chpl

```diff
@@ -411,6 +411,18 @@ proc max(type t) where isComplexType(t) {
   return x;
 }
 
+proc param_max(type t) param where t == int(8)
```

Comment:
Is there anyway to do something like 

proc param_max(type t:int(?w)) param { 
  return 1<<w-1:int(w); 
}